troubleshooting/ETW/Readme.md

```diff
@@ -139,55 +142,25 @@ with the following content:
 }
 ```
 
-To disable self-diagnostics, delete the above file.
+#### Configuration Parameters
+
+A `FileSize`-KiB log file named as `ExecutableName.ProcessId.log` (e.g. `foobar.exe.12345.log`) will be generated at the specified directory `LogDirectory`.
+
+1. `LogDirectory` is the directory where the output log file will be stored. 
+It can be an absolute path or a relative path to the current directory.
+
+2. `FileSize` is a positive integer, which specifies the log file size in [KiB](https://en.wikipedia.org/wiki/Kibibyte).
+This value must be between 1 MiB and 128 MiB (inclusive), or it will be rounded to the closest upper or lower limit.
+
+3. `LogLevel` is the lowest level of the events to be captured. 
+This value must match one of the [fields](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventlevel#fields) of the `EventLevel` enum.
+Lower severity levels encompass higher severity levels (e.g. `Warning` includes the `Error` and `Critical` levels).
+
+**Warning**: If the SDK fails to parse any of these fields, the configuration file will be treated as invalid and self-diagnostics will be disabled.
 
-Tip on finding the "current working directory":
```

Comment:
The tip might help some people who may be running the program by some server, and the "current working directory" might not be easy to find out.